src/admin-views/widgets/components/checkbox.php

```diff
@@ -11,28 +11,33 @@
  *
  * @version TBD
  *
- * @var string $label Label for the checkbox.
- * @var string $value Value for the checkbox.
- * @var string $id    ID of the checkbox.
- * @var string $name  Name attribute for the checkbox.
+ * @var string $label      Label for the checkbox.
+ * @var string $value      Value for the checkbox.
+ * @var string $id         ID of the checkbox.
+ * @var string $name       Name attribute for the checkbox.
+ * @var string $dependency The dependency attributes for the control wrapper.
  */
 
 ?>
-<p
-		class="tribe-widget-checkbox tribe-common-form-control-checkbox"
+<div
+		class="tribe-widget-form-control tribe-widget-form-control--checkbox"
+		<?php
+		// Not escaped - contains html ( data-attr="value")
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 71e295695ad73e60420b67e71c79dd4b</sub>